### PR TITLE
Create MemeCam.md

### PR DESCRIPTION
**Pull request by longxinzhang**
I add a new lesson to help the new Prompt Engineer learners to understand how to use GPT 3.5 to reverse-engineer an AI-driven app.

### CODE REVIEW

**Comment by trigaten** [comment]
This looks super cool. Here is the error:
```
  66 | <h1 {...{"id":"another-note"}}>{`Another Note`}</h1>
  67 | <p>{`The author Longxin Zhang is from find-my.ai. 芜湖～关注find my ai，有微信公众号和即刻哦！`}</p>
> 68 |     </MDXLayout>
     |                 ^
  69 |   )
  70 | };
  71 | MDXContent.isMDXComponent = true
  ```

File: docs/advanced_applications/MemeCam.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+Tahda! A (future) viral meme pic generated!
+
+<img width="300" alt="memecam" src="https://user-images.githubusercontent.com/30429387/231500025-bd63e1fc-2d6c-4c38-adf1-718cbabb65ce.png">
+
+Have you ever wondered how they did it?🤨
+
+Even though the author didn't reveal much infomation about the development of this app, as 
+AI enthusistic, we can try to **reverse-engineer** the development process based on the results.
+Which is the key of this lesson: if you believe an app used GPT-3.5 or GPT-4, you can write a 
+samilar prompt to reproduce someone's work.
```
Comment (trigaten) [inline docs/advanced_applications/MemeCam.md:19]: similar*

**Comment by longxinzhang** [comment]
thanks so much @trigaten . let me have a look at it. 

I am not so familiar with Github and markdown.

Give me some time.

File: docs/advanced_applications/MemeCam.md (hunk 1 above)
Comment (longxinzhang) [reply docs/advanced_applications/MemeCam.md:19]: thanks for finding the typo 
:-)

**Comment by trigaten** [comment]
Closing since you have another open PR with this content that is passing build.

**Pull request closed without merging**